docs/src/main/asciidoc/security-authentication-mechanisms.adoc

```diff
@@ -81,6 +81,69 @@ The resulting digest is used as a key for AES-256 encryption of the cookie value
 The cookie contains an expiry time as part of the encrypted value, so all nodes in the cluster must have their clocks synchronized.
 At one-minute intervals, a new cookie gets generated with an updated expiry time if the session is in use.
 
+To get started with form authentication, you should have similar settings as described in xref:security-basic-authentication-howto.adoc[Enable Basic authentication] and property `quarkus.http.auth.form.enabled` must be set to `true`.
+
+Simple application using form-base authentication can look similar to this:
+
+[source,java]
+----
+@Path("")
+public class FormResource {
+
+    @GET
+    @Path("hello")
+    @RolesAllowed("user")
+    public String hello(@Context SecurityContext security) {
+        return "Hello " + security.getUserPrincipal().getName();
+    }
+
+    @GET
```

Comment:
IMHO we can just drop this example, for the user facing application one would not return Strings but for example here produce HTML with Qute. This specific doc's goal is to give an overview of the authentication mechanisms only. 
Perhaps it is time to introduce a dedicated doc related to the form authentication

Reply:
Ok I'll remove it